File: CST-105/PolyStack/src/StackableInt.java

public class StackableInt extends StackableObject
{
	private int Data;
	
	public StackableInt(int data)
	{
		Data = data;
	}

	@Override
	public String GetData()
	{
		return String.valueOf(Data);
	}

	@Override
    public Boolean HasData()
    {
        return(true);
    }
}

File: CST-210/PolyStack/PolyStack/StackableInt.cpp
#include "StackableInt.h"

StackableInt::StackableInt(int data)
{
    m_data = data;
}

string StackableInt::GetData(void)
{
    return to_string(m_data);
}

bool StackableInt::HasData(void)
{
    return true;
}
